myhpom/templates/myhpom/upload/requirements.html

```diff
@@ -13,12 +13,134 @@
+				<ol class="upload-requirements-link__ol">
+					{% for link in requirement.staterequirementlink_set.all %}
+					<li class="upload-requirements-link__li">
+						<a href="{{ link.href }}" onClick="event.preventDefault();">{{ link.text }}</a>
+					</li>
+					{% endfor %}
+				</ol>
+			</li>
+			{% endfor %}
+		</ol>		
+	</div>
+</div>
+
+<div class="row">
+	<div class="col">
+		<h4 class="upload-requirements__h4">My file is valid starting:
+			<a class="upload-requirements__a--right" href="" onClick="event.preventDefault();">What is this?</a>
+		</h4>
+		<form id="id_requirements_form" class="upload-requirements__form" method="POST" action="{% url 'myhpom:upload_requirements' %}">
+			{% csrf_token %}
+			<div id="id_valid_date_div" class="upload-requirements__input-div{% if form.errors.valid_date %}--invalid{% endif %}">
+				<input id="id_valid_date" name="valid_date" data-min-year="{{ MIN_YEAR }}" data-max-year="{{ MAX_YEAR }}"{% if form.errors.valid_date %} class="upload-requirement__input--invalid"{% endif %}/>
+			</div>
+			<div id="id_valid_date_errors" class="upload-requirements__div-errors">
+				{{ form.errors.valid_date }}
+			</div>
+		</form>
+	</div>
+</div>
+
+<div class="row">
+	<div class="col">
+		<ol class="upload-requirements__ol">
+			<li class="upload-requirements__li">
+				<input type="checkbox" id="id_approval_checkbox" class="upload-requirements__checkbox"/>
+				<label class="upload-requirements__label" for="id_approval_checkbox">I approve this file for storage</label>
+				<ol class="upload-requirements-link__ol">
+					<li class="upload-requirements-link__li">
+						<a href="" onClick="event.preventDefault();">Review Terms and Conditions</a>
+					</li>
+				</ol>
+			</li>
+		</ol>
+	</div>
+</div>
+
 <div class="row">
     <div class="col">
-        <form action="{% url "myhpom:upload_sharing" %}">
-            <input type="file" class="form-control-file">
-            <button class="advance-directive-widget__button--primary" type="submit">Submit</button>
-        </form>
+        <input id="id_input_select_file" type="file" class="form-control-file"">
+        <button id="id_button_submit" class="advance-directive-widget__button--primary" type="submit">Submit</button>
     </div>
 </div>
+
+<div class="row">
+	<div class="col">
+		<a href="{% url 'myhpom:dashboard' %}">CANCEL</a>
+	</div>
+</div>
+
+<script>
+jQuery(function ($) {
+    var $requirements = $('.upload-requirements__checkbox');
+    var $submitButton = $('#id_button_submit');
+    var $directiveDateInput = $('#id_valid_date');
+    var $inputSelectFile = $("#id_input_select_file");
+
+	function submitButtonEnableDisable () {
+		console.log('file = ' + $inputSelectFile.val());
+		if ($('.upload-requirements__checkbox:checked').length == $requirements.length
+			&& $directiveDateInput.combodate('getValue') != ''
+			&& $inputSelectFile.val().trim() != ''
+		) {
+    		$submitButton.prop('disabled', null).css('cursor', 'pointer');
```

Comment:
Rather than setting properties on the button create two classes and add/remove the class as appropriate (say `.upload-requirements__submit` and `.upload-requirements__submit--disabled`). This keeps styling/behavior  cleanly separated.

Reply:
except that the disabled prop can't be fully emulated in css...? I want the pointer event to fire (and change the cursor) but not to allow the click. Otherwise, I can just add "pointer-events: none;" to the --disabled class.

Reply:
True, I've no issue with the disabled property. I'm concerned with setting css properties in jquery. Rather than setting individual css properties, use classes.